administrator/components/com_admin/sql/updates/mysql/4.1.0-2021-07-16.sql

```diff
@@ -0,0 +1,41 @@
+--
+-- Table structure for table `#__guidedtours`
+--
+
+CREATE TABLE IF NOT EXISTS `#__guidedtours` (
+  `id` int NOT NULL AUTO_INCREMENT PRIMARY KEY,
+  `description` text NOT NULL,
+  `title` varchar(255) NOT NULL,
+  `date` DATE NOT NULL DEFAULT CURRENT_TIMESTAMP ,
```

Comment:
```suggestion
  `date` datetime NOT NULL DEFAULT CURRENT_TIMESTAMP ,
```
There is no data type "date" in MySQL, Same mistake in all other SQL here. You should be able to find such errors by testing your statements in phpMyAdmin.

Reply:
Because the same error in installation/sql/mysql/extensions.sql makes installation fail with an SQL syntax error, the system tests fail.

Reply:
sorry, my mistake, there is a date data type .. but we should not use it in Joomla for other reasons. there must be some other mistake in the SQL here which makes it fail.